hover.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hover nav</title>
+</head>
+
+<body>
+    <header class="page-header">
+        <a class="main-logo"><img src="img/logo.svg" alt="binabox-logo">
+            <span class="logo-text">BINABOX</span></a>
+        <nav class="top-nav">
+            <ul class="nav-list">
+                <li class="nav-item">
+                    <div class="dropdown">
+                        <button class="dropbtn">HOME
+                            <i class="fa fa-caret-down"></i>
+                        </button>
+                        <div class="dropdown-content">
```

Comment:
Homogeneous repeating elements (navigation; tags perceived as a list) should be implemented as a list - http://htmlbook.ru/html/ul